<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>835</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1275</width>
    <height>835</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1275</width>
    <height>835</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>201</width>
     <height>671</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>643</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pb_findMusic">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>发现音乐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_podcast">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>播客</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_vedio">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>视频</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_friend">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>朋友</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_onlineLive">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>直播</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_privateFM">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>私人FM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_myMusic">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background:transparent;</string>
       </property>
       <property name="text">
        <string>我的音乐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_localDownload">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>本地与下载</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_recentPlayer">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>最近播放</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_myCloudStorage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>我的音乐云盘</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_myPodcast">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>我的播客</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_myCollect">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>我的收藏</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>1281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pb_lastMusic">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/bfzn_prev.png</normaloff>:/images/bfzn_prev.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_nextMusic">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/bfzn_next.png</normaloff>:/images/bfzn_next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_pauseAndPlayer">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/bfzn_play.png</normaloff>:/images/bfzn_play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_circulationStyle">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/orderPlayer.png</normaloff>:/images/orderPlayer.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_singer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_musicName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="slider_music">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_volume">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>0</y>
      <width>22</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_musicTime">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_volume">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>70</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/voice_open.png</normaloff>:/images/voice_open.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_lrc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/lrc.jpg</normaloff>:/images/lrc.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>91</width>
      <height>91</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pb_lastMusic</zorder>
   <zorder>pb_pauseAndPlayer</zorder>
   <zorder>pb_circulationStyle</zorder>
   <zorder>lb_singer</zorder>
   <zorder>lb_musicName</zorder>
   <zorder>slider_music</zorder>
   <zorder>slider_volume</zorder>
   <zorder>lb_musicTime</zorder>
   <zorder>pb_volume</zorder>
   <zorder>pb_lrc</zorder>
   <zorder>pb_nextMusic</zorder>
  </widget>
  <widget class="QListWidget" name="lw_musicList">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>1081</width>
     <height>481</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_downloadManagement">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>下载管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_localMusic">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>本地音乐</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_hasDownloadMusic">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>已下载单曲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_hasDownloadVoice">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>已下载声音</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_hasDownloadMV">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>已下载MV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_beingDownload">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>正在下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_playerAll">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>播放全部</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_searchMyDownloadMysic">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>194</y>
     <width>211</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_searchMyDownloadMusic">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>200</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_addMusic">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>添加歌曲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_delMusic">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>删除歌曲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_playMV">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>播放MV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_headBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(236, 65, 65);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_returnLast">
   <property name="geometry">
    <rect>
     <x>253</x>
     <y>16</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_identifyMysic">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>16</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_returnNext">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>16</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>6</y>
     <width>201</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lb_playerIcon">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="src.qrc">:/images/topicon.bmp</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_playerName">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>网易云音乐</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="le_searchMusic">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>6</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_searchMusic">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_userLogin">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_reginster">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
